packages/mdc-snackbar/index.js

```diff
@@ -21,4 +21,196 @@
  * THE SOFTWARE.
  */
 
-export class MDCSnackbar {}
+import {MDCComponent} from '@material/base/index';
+import MDCSnackbarFoundation from './foundation';
+import {getCorrectEventName} from '@material/animation/index';
+import {strings} from './constants';
+import * as ponyfill from '@material/dom/ponyfill';
+
+class MDCSnackbar extends MDCComponent {
+  static attachTo(root) {
+    return new MDCSnackbar(root);
+  }
+
+  initialSyncWithDOM() {
+    const {SURFACE_SELECTOR} = MDCSnackbarFoundation.strings;
+
+    /** @type {!HTMLElement} */
+    this.surfaceEl_ = this.root_.querySelector(SURFACE_SELECTOR);
```

Comment:
I suspect these members will first need to have types declared in `constructor` in order for closure tests to pass.

Reply:
Done